examples/basics/UI.py

```diff
@@ -0,0 +1,230 @@
+import gradio as gr
+
+imd_runner = None
+
+
+def run_simulation(
+    simulation_type,
+    input_files,
+    trajectory_files,
+    state_file,
+    verbosity,
+    show_progression,
+    server_name,
+    port,
+    simulation_fps,
+    frame_interval,
+    start_paused,
+    include_velocities,
+    include_forces,
+    record_trajectory,
+    trajectory_output_file,
+    shared_state_file,
+):
+    """
+    Runs a simulation with the given parameters.
+
+    :param simulation_type: choose between "From xml" and "From recording"
+    :param input_files: list of input files to run as live simulation
+    :param trajectory_files: list of trajectory files to run as playback simulation
+    :param state_file: list of state files to run as playback simulation
+    :param verbosity: choose between "Normal", "Verbose", "Super verbose"
+    :param show_progression: show simulation progression
```

Comment:
This doesn't seem to do anything currently -- if it's easy to have it match rust (periodically print out current frame etc) then may as well do it, otherwise I don't think it's very important right now?

Reply:
I've removed this for now.